### ISSUE TITLE
Zu viel Whitespace

### ISSUE THREAD

**Issue by frankgerhardt** [mitfahrverband/iframe#24]
![image](https://user-images.githubusercontent.com/550621/166650237-8a8669f3-2d34-4555-a59b-de275ffc4bd5.png)
Die Tabs sind super :-) Aber der Ort wird jetzt abgekürzt in "Dü.", mit CSS ich weiss. Das Kürzen mit CSS ist gut, aber an dieser Stelle kontraproduktiv. Gefühlt ist mehr als die Hälfte des Platzes Whitespace. 
- der Platz sollte voll genutzt werden, so dass wenig Whitespace entsteht
- wenn gar nicht so viel Platz gebraucht wird, sollte der iframe insgesamt kleiner werden (meine Rede die ganze Zeit ;-) ) Schon allein wegen der #22 Mobil-Tauglichkeit sollte er kleiner sein.

**Comment by MartinH-open** [comment]
"designer lieben white space" - alles so schön freundlich hell hier .. ;)

Aktuell ist das Kürzen nicht dynamisch mit CSS gemacht, sondern hardcoded in PHP bei der Datenerfassung
für Düsseldorf => Dü.  und Hauptbahnhof => Hbf  usw.

Das iframe wird ja von außen statisch auf eine Größe (in px) definiert.
Ansonsten skaliert die Anzeige drinnen schon in bestimmten Grenzen.

Eine dynamische Kürzung von Textteilen (per CSS) wäre eine neue Baustelle. 

Ansonsten brauchen wir hier besser eine dynamische iframe Einbindung in % der Seitebreite: width="100%"
`<iframe src="https://ridewithukraine.eu/iframedemo.php" height="840" width="100%"></iframe>`
Damit sieht unser Beispiel https://ridewithukraine.eu/iframe-beispiel.html jetzt so aus auf einem 412 px breiten Smartphone:

![image](https://user-images.githubusercontent.com/1383111/166669417-d551a908-24af-4eb6-b86d-d706a361ca9c.png)

**Comment by frankgerhardt** [comment]
> Eine dynamische Kürzung von Textteilen (per CSS) wäre eine neue Baustelle.

In der alleresten Verison des iframe, die meine Studis gemacht haben, war das drin, mit CSS. Ich weiss das, weil sie damit viel Zeit verbraten haben. Der Vorteil ist, dass die Daten "Düsseldorf" enhalten und je nach Breite der Anzeige kürzt CSS automatisch mit "...". Wenn du es so machst, hast du auch "Dü." in einer breiten Anzeige.

**Comment by frankgerhardt** [comment]
@MartinH-open vergleich mal oben die Schriftgröße von Lori ipsum außerhalb und mit der Größe im iframe. Im iframe ist es viel zu groß.

**Comment by MartinH-open** [comment]
Ja, ich finde die Schriftgröße im iframe auch viel zu groß - aber das ist alles so wie bisher unter ridewithukraine.eu auch definiert und genutzt.
Man kann da schon was einschrumpfen: font-size, line-height usw  aber das hat Einfluß auf die Perlenschnur etc. 
Da gibt es einige Konsequenzen, die man beachten müsste.

Aber so etwa könnte es schon aussehen, wenn man es kleiner haben will:

![image](https://user-images.githubusercontent.com/1383111/166704953-abbd87ba-ffd8-44ec-b2ff-be447f2eb5d1.png)

**Comment by frankgerhardt** [comment]
Wenn die Demo rum ist, als Nächstes ridewithukraine als iframe!

**Comment by MartinH-open** [comment]
ridewithukraine nutzt auch jetzt schon intern einen iframe.
Gemeint ist hier wohl, dass die Darstellung kompakter und es einen weiteren Reiter im iframe geben sollte, der das "Anbieten" einer Mitfahrt innerhalb des iframes erlaubt. Das wäre klar ein neues Enhancement.
